packages/cfd/src/Containers/props.types.ts

```diff
@@ -61,3 +62,64 @@ export type TCFDResetPasswordModal = {
     setCFDPasswordResetModal: (value: boolean) => void;
     history: History;
 };
+
+export type TFormValues = {
```

Comment:
@hirad-rewok  you can do it shorter
check please this article about feature 'Record' in TS:
https://www.typescriptlang.org/docs/handbook/utility-types.html
we can use it to avoid repeating when all values of all keys are the same   
```suggestion
export type TFormValues = Record<'old_password' | 'new_password' | 'password_type', string>
```